types/belter/index.d.ts

```diff
@@ -19,7 +19,7 @@ export function isIos(ua: string): boolean;
 export function isGoogleSearchApp(ua: string): boolean;
 export function isQQBrowser(ua: string): boolean;
 export function isIosWebview(ua: string): boolean;
-export function isAndroidWebview(ua?: string): boolean;
+export function isAndroidWebview(ua: string): boolean;
```

Comment:
It looks like it *is* optional? https://github.com/krakenjs/belter/blob/cdddc4f8ddb172d29db9e7e1ad1eeeacfb93e215/src/device.js#L68

Reply:
Yes, indeed! Thanks for the review!

Reply:
I'm confused - I thought you had agreed that it should remain optional, so I was waiting for an update to the PR before signing off.  If you actually meant that is should _not_ be optional, can you please explain why not?